apps/design/frontend/src/geography_screen.test.tsx

```diff
@@ -440,6 +450,43 @@ describe('Precincts tab', () => {
       })
     );
 
+    // Update NH-configurable fields
+    const split3ElectionTitleOverrideInput = within(split3Card).getByLabelText(
+      'Election Title Override'
+    );
+    expect(split3ElectionTitleOverrideInput).toHaveValue('');
+    userEvent.type(
+      split3ElectionTitleOverrideInput,
+      assertDefined(changedPrecinct.splits[1].electionTitleOverride)
+    );
+
+    const split3ClerkSignatureCaption = within(split3Card).getByLabelText(
+      'Clerk Signature Caption'
+    );
+    expect(split3ClerkSignatureCaption).toHaveValue('');
+    userEvent.type(
+      split3ClerkSignatureCaption,
+      assertDefined(changedPrecinct.splits[1].clerkSignatureCaption)
+    );
+
+    const signatureInput =
+      within(split3Card).getByLabelText('Upload Image').parentElement!;
+    userEvent.upload(
+      // within(signatureInput).getByLabelText('Upload Image'),
+      signatureInput,
+      new File([dummyImage], 'new_seal.svg', {
+        type: 'image/svg+xml',
+      })
+    );
+    await waitFor(() =>
+      expect(within(split3Card).getByRole('img')).toHaveAttribute(
+        'src',
+        `data:image/svg+xml;base64,${Buffer.from(dummyImage).toString(
```

Comment:
Why are you encoding as base64 again when it's already base64? Either make `dummyImage` not base64 or just interpolate `${dummyImage}` as-is here.

Reply:
Thanks - [ac7435f](https://github.com/votingworks/vxsuite/pull/5823/commits/ac7435f2bd884d7f04b7b43aa31185582d1f020e)